tools/clang/unittests/HLSL/MSFileSysTest.cpp

```diff
@@ -146,12 +146,29 @@ class MockDxcSystemAccess : public IDxcSystemAccess
   }
 
   virtual HRESULT STDMETHODCALLTYPE EnumFiles(LPCWSTR fileName, IEnumSTATSTG** pResult) override {
-    STATSTG items[] =
-    {
-      { L"filename.hlsl", STGTY_STREAM, { 0, 0 }, { 0, 0 }, { 0, 0 }, { 0, 0 }, 0, 0, GUID_NULL, 0, 0 },
-      { L"filename2.fx",  STGTY_STREAM, { 0, 0 }, { 0, 0 }, { 0, 0 }, { 0, 0 }, 0, 0, GUID_NULL, 0, 0 }
-    };
-    unsigned testCount = (unsigned)_countof(items);
+    STATSTG items[] = {{const_cast<wchar_t *>(L"filename.hlsl"),
```

Comment:
It's unfortunate that the filename in STATSTG isn't const. I don't imagine any of the usages will change that string. If they do, trying to change a literal would be very bad. I would still prefer creating a local editable allocation initialized to this string value rather than passing in a literal as non-const.

Reply:
Fixed.